samples/mapzen-android-sdk-sample/src/main/java/com/mapzen/android/sdk/sample/CustomMarkerActivity.java

```diff
@@ -0,0 +1,65 @@
+package com.mapzen.android.sdk.sample;
+
+import com.mapzen.android.graphics.MapFragment;
+import com.mapzen.android.graphics.MapzenMap;
+import com.mapzen.android.graphics.MarkerPickListener;
+import com.mapzen.android.graphics.OnMapReadyCallback;
+import com.mapzen.android.graphics.model.BitmapMarker;
+import com.mapzen.android.graphics.model.MarkerOptions;
+import com.mapzen.tangram.LngLat;
+
+import android.os.Bundle;
+import android.view.View;
+import android.widget.Toast;
+
+/**
+ * Custom marker demo.
+ */
+public class CustomMarkerActivity extends BaseDemoActivity implements MarkerPickListener {
+
+  private MapzenMap map;
+  private BitmapMarker bitmapMarker;
+
+  @Override protected void onCreate(Bundle savedInstanceState) {
+    super.onCreate(savedInstanceState);
+    setContentView(R.layout.activity_custom_marker);
+    final MapFragment mapFragment =
+        (MapFragment) getSupportFragmentManager().findFragmentById(R.id.fragment);
+    mapFragment.getMapAsync(new OnMapReadyCallback() {
+      @Override public void onMapReady(MapzenMap map) {
+        CustomMarkerActivity.this.map = map;
+        configureMap();
+      }
+    });
+  }
+
+  private void configureMap() {
+    map.setMyLocationEnabled(true);
+    map.setCompassButtonEnabled(true);
+    map.setPersistMapState(true);
+    map.setPosition(new LngLat(-73.985428, 40.748817));
+    map.setZoom(16);
+    map.setMarkerPickListener(this);
+
+    findViewById(R.id.add_marker_btn).setOnClickListener(new View.OnClickListener() {
+      @Override public void onClick(View v) {
+        final MarkerOptions markerOptions = new MarkerOptions()
+            .position(new LngLat(-73.985428, 40.748817))
```

Comment:
Agree. When I see a sweet my location button on the screen I really want to press it! Then I don't see the custom marker :( which kind of defeats the purpose of the demo.

I disabled my location in this example. This also has the nice side effect of fixing the crash on rotation ;)